1.5/Defs/Buildings/Buildings_Structure.xml

```diff
@@ -340,4 +340,147 @@
             <li Class="PawnStorages.PSExtension" />
         </modExtensions>
     </ThingDef>
+
+    <ThingDef ParentName="BuildingBase">
+        <defName>PS_BugWallStorage</defName>
+        <label>Wall Nest</label>
+        <description>An insect like wall nest capable of storing multiple pawns. Can be placed on walls like a Wall Lamp.</description>
+        <thingClass>PawnStorages.Building_PawnStorage</thingClass>
+        <rotatable>true</rotatable>
+        <graphicData>
+            <texPath>Things/Building/PS_BugWallStorage</texPath>
+            <graphicClass>Graphic_Multi</graphicClass>
+            <drawOffsetNorth>(0,0,0.75)</drawOffsetNorth>
+            <drawOffsetSouth>(0,0,-0.9)</drawOffsetSouth>
+            <drawOffsetWest>(-0.9,0,0)</drawOffsetWest>
+            <drawOffsetEast>(0.9,0,0)</drawOffsetEast>
+            <damageData>
+                <enabled>false</enabled>
+            </damageData>
+        </graphicData>
+        <tickerType>Normal</tickerType>
+        <altitudeLayer>BuildingOnTop</altitudeLayer>
+        <passability>Standable</passability>
+        <terrainAffordanceNeeded IsNull="True" />
+        <researchPrerequisites IsNull="True" />
+        <pathCost>0</pathCost>
+        <overrideMinifiedRot>North</overrideMinifiedRot>
+        <minifiedDrawOffset>(0, 0, 0.2)</minifiedDrawOffset>
+        <uiOrder>2991</uiOrder>
+        <containedPawnsSelectable>true</containedPawnsSelectable>
+        <building>
+            <isEdifice>false</isEdifice>
+            <isAttachment>true</isAttachment>
+        </building>
+        <placeWorkers>
+            <li>Placeworker_AttachedToWall</li>
+        </placeWorkers>
+        <inspectorTabs>
+            <li>PawnStorages.TickedStorage.ITab_PawnList</li>
+        </inspectorTabs>
+        <designationCategory>Furniture</designationCategory>
+        <comps>
+            <li Class="PawnStorages.CompProperties_PSAssignableToPawn">
+                <drawAssignmentOverlay>false</drawAssignmentOverlay>
+                <maxAssignedPawnsCount>16</maxAssignedPawnsCount>
+                <colonyAnimalsOnly>true</colonyAnimalsOnly>
+                <showGizmo>true</showGizmo>
+            </li>
+            <li Class="PawnStorages.TickedStorage.CompProperties_TickedStorage">
+                <TickAge>false</TickAge>
+                <TickHediffs>false</TickHediffs>
```

Comment:
have one for a single pawn too, prisoner that shows head with hediff ticking and lower case props please :)

Reply:
The wall nest doesn't tick hediffs, that's what the cocoon does. There's a wall mountable and free standing cocoon, both show the pawn. Showing just the head was too much work, so went with drawing the full pawn and then drawing tendrils over them "holding" them in place. 

Props lowercased.